src/conn/conn_api.c

```diff
@@ -2290,6 +2290,31 @@ __wti_heuristic_controls_config(WT_SESSION_IMPL *session, const char *cfg[])
     return (0);
 }
 
+/*
+ * __wti_cache_eviction_controls_config --
+ *     Set cache_eviction_controls configuration.
+ */
+int
+__wti_cache_eviction_controls_config(WT_SESSION_IMPL *session, const char *cfg[])
+{
+    WT_CONFIG_ITEM cval;
+    WT_CONNECTION_IMPL *conn;
+    uint64_t flags;
+
+    conn = S2C(session);
+
+    WT_RET(__wt_config_gets(session, cfg, "cache_eviction_controls.flags", &cval));
+    flags = (uint64_t)cval.val;
+
+    if ((flags & WT_CACHE_EVICT_INCREMENTAL_APP) != 0)
+        F_SET(&conn->cache_eviction_controls, WT_CACHE_EVICT_INCREMENTAL_APP);
+
+    if ((flags & WT_CACHE_EVICT_SCRUB_UNDER_TARGET) != 0)
```

Comment:
I think it is better to have a if (flags) and put both the settings under from readability view, to indicate sub controls are activated only if cache_eviction_controls are activated.